lib/rules/base.js

```diff
@@ -65,6 +68,26 @@ module.exports = class Base {
     this.source = options.rawSource.match(reLines);
   }
 
+  get filePath() {
+    return this._filePath;
+  }
+
+  get resolvedFilePath() {
+    return path.resolve(this.workingDir, this.filePath);
+  }
```

Comment:
Thoughts on naming this `absoluteFilePath`?

Reply:
@rwjblue in my understanding, `filePath` already absolute one, nope?

Reply:
for now, removed from public surface

Reply:
If `filePath` is already absolute, what is the point of the `path.resolve` here? Do you mean to find the relative file path instead (e.g. from `/Users/rwjblue/src/app-name/app/templates/foo.hbs` to `app/templates/foo.hbs`)?

Reply:
@rwjblue yes, it's my goal, but I don't know how to get it working (see `things to figure out` in mr description, because of cwd)

yeah, I suppose filePath  should be absoulte.

https://github.com/ember-template-lint/ember-template-lint/issues/1346#issuecomment-631720981
![image](https://user-images.githubusercontent.com/1360552/106923472-28239600-671f-11eb-928d-b343f35bad3b.png)

Reply:
We can confirm (and add a `relativeFilePath` or something) in a separate PR, since it isn't _really_ related to this anymore (now that you removed it).

Reply:
agree, `resolvedFilePath` removed from changes